src/LivewireServiceProvider.php

```diff
@@ -92,6 +97,9 @@ public function registerCommands()
         if ($this->app->runningInConsole()) {
             $this->commands([
                 LivewireMakeCommand::class,
```

Comment:
Use these as the base -> extended class
'make' => 'touch'
'move' => 'mv'
'copy' => 'cp'
'delete' => 'rm'



Hook into command suggestions and see if we can correct "create" -> "make"

Reply:
Done.

I'm not sure about the suggestions bit, but I did manage to hide the secondary aliases so that they do not clutter up the `artisan list`